### ISSUE TITLE
CW and <https://gitter.im/Codewars/codewars.com> need a moderator

### ISSUE THREAD

**Issue by g964** [codewars/codewars.com#2133]
@kazk
Somebody raised an issue there
<https://www.codewars.com/kata/5df754981f177f0032259090/discuss/>

I answered it and what was my astonishment to read on gitter.im/Codewars the following words

"
also I wish g964 can be less dumb...

bruh, can g964 stop accusing others for falsely raising issues on their katas before even checking the test inputs?
a typical discourse with g964 is like, they won't even bother checking  the thing being pointed out and require you to ELI5 the entire thing to them or else they'll automatically treat you as raising a false issue

dude, I want to grab a warhammer and maul g964 so much right now talking about how dumb can someone be
"

The last words seems particularly grave. As far as I can understand it's a threat to my life. It is not worthy of a man who, one could suppose it, is smart and well brought up.

I know that gitter.im/Codewars is not in your part but you can understand that it is unbearable to read this kind of words from human beings and intended for other human beings. You have given - it seems to me - great powers to some but I can't suppose you authorize those to brutalize the others.

Maybe a team of real moderators could remedy to that state that is worse from day to day.

**Comment by kazk** [comment]
@g964 I'm sorry about what happened. His behavior was not unacceptable and I posted a comment on Gitter about it.

You didn't deserve those words.
He said he was frustrated that you frequently closed the issue without investigating.
I don't think I see this problem in the discussion link you posted, but I've seen you closing issues immediately saying "Not an issue of the kata. X people passed". Maybe instead ask the user for the details (e.g., examples of the input and the expected output) before closing it and investigate if they get back to you. The reply you often use when closing the issue can be wrongly interpreted.

---

It's been difficult to manage the community because I'm the only Codewars dev/admin and any tasks related to Qualified have higher priority so I haven't had the time to be looking around or make fundamental changes.
I often see some miscommunication happening (we all come from different backgrounds and speak different languages so it can be difficult). I hope the Codewars community can be more laid-back and assume best intent in general.

> Maybe a team of real moderators could remedy to that state that is worse from day to day.

I have plans to implement ways to delegate some admin tasks to the community. It's delayed because I've been busy with tasks unrelated to Codewars, but I still want to do it soon.
To find the users we can trust, I'm planning to create a community team on GitHub and start from asking some of the community join the team to help me triage issues on GitHub and see how they behave and _communicate_. If that works well, I'll start giving the team members extra privileges on Codewars. Giving some users a privilege on Codewars to fix kata was also an experiment that was easy to start.

**Comment by kazk** [comment]
Closing because we have moderators now. We'll try our best to keep our Discord safe and welcoming.
